### ISSUE TITLE
help text

### ISSUE THREAD

**Issue by denispelli** [BrainardLab/mQUESTPlus#12]
dear david
the help text for QUESTPlus is not very helpful. A beginner needs two things: a link to beau's article (or some other tutorial) and a good example of code to copy and modify. In fact that's what qpRun is, but I would never guess that from what I see when I type:
>> help questplus
  mQUESTPlus - questplus
 
  The core QUEST+ routines.
 
  qpFit                         - Maximum likelihood fit of a psychometric function to a trial data array.
  qpInitialize                  - Initialize the questData structure for a QUEST+ experiment.
  qpParams                      - Set user defined parameters for a QUEST+ run.  Called by qpInitialize.
  qpQuery                       - Use questData structure to get next recommended stimulus index and stimulus
  qpRun                         - High level function that runs a QUEST+ experiment
  qpUpdate                      - Update the questData structure for the trial stimulus and outcome.
  qpUpdateUpdateExpectedNextEnropiesByStim - Update the table of expected next entropies for each stimulus.

I think i've mentioned before that I thought users were expected to write their own code to call qpRun. I couldn't tell that it was meant to be an example. I would favor renaming "qpRun" to "qpExample", but at a minimum the help text should be changed. Change:
- High level function that runs a QUEST+ experiment
to
- An example user function to run a QUEST+ experiment

best
denis

**Comment by denispelli** [comment]
It would be great to add a link to Beau's JoV article to the help text for "help QuestPlus".

Andrew B. Watson; QUEST+: A general multidimensional Bayesian adaptive psychometric method. Journal of Vision 2017;17(3):10. 
http://jov.arvojournals.org/article.aspx?articleid=2611972

**Comment by DavidBrainard** [comment]
Made a bunch of changes to try to help with this.

What you really want, for your purposes, is qpQuestPlusCoreFunctionDemo, not the source
to qpRun.  I have tried to make this clearer in a number of places.

It is an open question, to me at least, as to whether in general people will want to let qpRun run their experiment or have their experiment call the QUEST+ routines. I'm like you and call the routines.  But the paper is organized around qpRun and I think there is enough value to following the paper that I'm going to keep its name as is.  The paper also discusses the alternative approach.

Closing.  Reopen if you have more suggestions along these lines.
